src/Config/CliBuilder.ts

```diff
@@ -169,4 +245,8 @@ export class CliBuilder {
       }
     });
   }
+
+  private handleDeprecatedOptions(_argv: Arguments) {
+    // Handle deprecated options here
```

Comment:
is it intentionally or are we going to implement in this PR ?

Reply:
Agree - i will remove.